packages/cardhost/app/templates/components/card-renderer.hbs

```diff
@@ -8,78 +8,83 @@ Ember Concurrency into the rendering of the card component here.
-                <DropZone
-                  @dropField={{@dropField}}
-                  @position={{0}}
-                  @toggleStubField={{this.toggleStubField}}
-                />
-              {{/if}}
-              <Isolated
-                @card={{card}}
-                @fields={{this.fields}}
+      <AnimatedContainer>
+        <section
+          class="card-renderer-isolated
+            {{this.mode}}
+            {{safe-css-string @card.adoptedFromName}}
+            {{safe-css-string @card.name}}
+            {{if (and this.cardstackSession.isAuthenticated @cardSelected) "selected"}}
+            {{concat "field-count-" this.fields.length}}
+            {{@class}}
+          "
+          tabindex="0"
+          data-test-card-renderer-isolated
+          data-test-card-is-dirty={{if @card.isDirty "yes" "no"}}
+          {{on "focus" (action this.cardIsFocused true)}}
+          {{did-insert this.focusCard true}}
+          {{did-update this.updateFields @card.isolatedFields}}
+        >
+          {{#if (eq @mode "view")}}
+            {{#animated-value @card watch="isSelected" use=this.headerAnimation group="header"}}
+              <CardRendererHeader
+                @cardSelected={{if (and this.cardstackSession.isAuthenticated @cardSelected) true}}
+                @card={{@card}}
+                @buttonRoute="cards.card.edit.fields"
+                @buttonText="Edit"
                 @mode={{this.mode}}
-                @setFieldValue={{@setFieldValue}}
-                @setFieldName={{@setFieldName}}
-                @removeField={{@removeField}}
-                @toggleStubField={{this.toggleStubField}}
-                @setNeededWhenEmbedded={{@setNeededWhenEmbedded}}
-                @setPosition={{@setPosition}}
-                @selectField={{@selectField}}
-                @selectedField={{@selectedField}}
-                @dropField={{@dropField}}
+                @contextMenuOpen={{@contextMenuOpen}}
+                @setContextMenu={{@setContextMenu}}
               />
-            </div>
-          </MaybeAnimate>
-        </div>
-      </section>
+            {{/animated-value}}
+          {{else}}
+            <AnimatedContainer>
+              {{#animated-value @card watch=this.mode use=this.headerAnimation group="header"}}
+                <CardRendererHeader
+                  @cardSelected={{true}}
+                  @card={{@card}}
+                  @buttonRoute="cards.card.view"
+                  @buttonText="Done Editing"
+                  @mode={{this.mode}}
+                  @contextMenuOpen={{@contextMenuOpen}}
+                  @setContextMenu={{@setContextMenu}}
+                />
+              {{/animated-value}}
+            </AnimatedContainer>
+          {{/if}}
+          <AnimatedContainer class="card-renderer-isolated--content">
+            {{#animated-value @card watch=(if (eq this.mode "view") "isSelected" this.mode) use=this.cardTransition group="body"}}
```

Comment:
same as the above in terms of watching `this.mode`, you actually end up watching a property on `@card` that `this.mode` evaluates to: `@card.view` or `@card.edit` or `@card.schema`, etc